python_modules/dagster/dagster/_core/definitions/result.py

```diff
@@ -16,6 +19,8 @@ class MaterializeResult(
         [
             ("asset_key", PublicAttr[Optional[AssetKey]]),
             ("metadata", PublicAttr[Optional[MetadataUserInput]]),
+            ("data_version", PublicAttr[Optional[DataVersion]]),
```

Comment:
let's list `data_version` after `check_results`. I think we should consider that feature "more important"